### ISSUE TITLE
Provide CLI feedback for plug-in installation

### ISSUE THREAD

**Issue by infotexture** [dita-ot/dita-ot#3109]
## Description

The `dita --install` option currently provides no confirmation when plug-ins are successfully installed (or removed). 

This is consistent with the typical "no news is good news" approach for basic UNIX utilities, and tolerable enough when installing or removing a single plug-in.

However, the current behavior is less helpful in situations when the `plugins` directory is manually modified to add/remove multiple plug-ins at once, ~or install multiple plug-ins from a single .zip file~.

## Possible Solution

Since DITA-OT maintains a list of currently installed plug-ins and updates this configuration during the integration process, it would be helpful to compare the before/after states and print a summary to the console upon completion, for example:

```console
$ dita --install
+ Installed com.example.plugin1
- Removed com.example.plugin2
```

It might also be useful to conclude the integration process by printing the current list of available plug-ins to the console via the `dita --plugins` option. 

Since the plug-in registry changes for #3028 also permit the installation of plug-ins by alias, this would also serve to confirm the user's intent by displaying the plug-in ID that was actually installed via the alias.

This would be more consistent with the feedback provided by other CLI-based package managers.

_(See related enhancement #3070)._

**Comment by jelovirt** [comment]
>  install multiple plug-ins from a single .zip file

Does that work? 🤔

**Comment by infotexture** [comment]
My recollection was that multiple plug-ins could be installed simultaneously by passing a .zip file that contained several plug-ins, each in top-level folders of the archive.

But I tested now, and only 1 of the 4 plug-ins in the archive was installed _(first in alphabetical order)_.

@robander Did I dream this, or did it used to work that way? 

Having to unpack the archive to the `plugins` folder first seems like an unnecessary step.

**Comment by robander** [comment]
I think I've asked about this in the past but it was never supported - like @jelovirt I was a bit surprised by that in the initial report. It came up for me when first working with some of the D4P and SVG Diagram support, each of which required a bundle of plugins to work. I think I asked about that at the time and it was clearly out of scope for the initial `dita --install` support, but I hadn't thought about it since.

**Comment by infotexture** [comment]
Resolved in #3110 & a17143c.
